src/Utilities/Compiler/DiagnosticDescriptorHelper.cs

```diff
@@ -33,7 +33,7 @@ internal static partial class DiagnosticDescriptorHelper
             var (defaultSeverity, enabledByDefault) = GetDefaultSeverityAndEnabledByDefault(ruleLevel, isEnabledByDefaultInFxCopAnalyzers);
 
 #pragma warning disable CA1308 // Normalize strings to uppercase - use lower case ID in help link
-            var helpLink = $"https://docs.microsoft.com/visualstudio/code-quality/{id.ToLowerInvariant()}";
+            var helpLink = $"https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/{id.ToLowerInvariant()}";
```

Comment:
This is the only manual change in this PR. Rest all are automatic updates from repo tooling on running pack.